compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeSceneLayer.skiko.kt

```diff
@@ -172,27 +169,24 @@ internal fun rememberComposeSceneLayer(
     val density = LocalDensity.current
     val layoutDirection = LocalLayoutDirection.current
     val parentComposition = rememberCompositionContext()
-    val compositionLocalContext by rememberUpdatedState(currentCompositionLocalContext)
+    val compositionLocalContext = currentCompositionLocalContext
     val layer = remember {
         scene.createLayer(
             density = density,
             layoutDirection = layoutDirection,
             focusable = focusable,
             compositionContext = parentComposition,
-        ).also {
-            it.compositionLocalContext = compositionLocalContext
-        }
+        )
     }
     layer.focusable = focusable
+    layer.compositionLocalContext = compositionLocalContext
```

Comment:
I assume this should be removed, if we don't set it in a layer. Is it correct, @MatkovIvan ?

Reply:
It shouldn't (we discussed)